tools/cla_check.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
```

Comment:
Putting this in a `tools` directory at the root implies it's a tool for development when it's really only for our GitHub Actions.

How about we put this in `.github/workflows/tools/cla_check.py` rather.

Reply:
Makes sense, i'll make it so!